contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -100,20 +100,27 @@ def _clone_repo(temp_dir, repo_url, verifyssl=True, ref='master'):
         # because we want the user to work with the repo in the
         # future.
         repo = Repo.clone_from(repo_url, temp_dir)
+        active_branch = repo.active_branch
```

Comment:
We always clone the repo so this shouldn't matter, right?

And that's really what we are interested in - repo's active branch (which happens to be repo's default remote branch as well).